File: dataset/human/python/file_1281.py
def main():
    n = int(input())
    a = list(map(int,input().split()))
    x = a[0]
    y = sum(a) - x
    x_y = abs(x - y)
    for i in range(1, n - 1):
        x += a[i]
        y -= a[i]
        x_y = min(x_y, abs(x - y))
    print(x_y)

if __name__ == '__main__':
    main()

File: dataset/human/java/file_1281.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] A = new int[n];
        long sum = 0;
        long res = Long.MAX_VALUE;
        for(int i = 0; i < n; i++) {
            A[i] = Integer.parseInt(sc.next());
            sum += A[i];
        }
        long tmp = 0;
        for(int i = 0; i < n-1; i++) {
            tmp += A[i];
            res = Math.min(res, Math.abs(2 * tmp - sum));
        }
        System.out.println(res);
    }
}
